src/views/penalty-details.njk

```diff
@@ -54,6 +55,21 @@
             autocomplete: "off"
           }) 
         }}
+        <div class='govuk-form-group'>
+          {% set detailsHTML %}
+          <p>
+            Your company number and reference number are shown at the bottom of your penalty notice.
+          </p>
+          <img src='{{ ROOT_URI }}/images/penalty-notice-image.png' class='call-out-image' alt=''/>
+          {% endset -%}
```

Comment:
HTML components should remain as single quoutes

Reply:
double.. 👍